### PR TITLE
Rc 906 more tweaks to connectd mp configure

### PR DESCRIPTION
**Pull request by Digital-Larry**

### CODE REVIEW

File: connectd/usr/bin/connectd_control [hunk 1]
```diff
@@ -857,7 +881,7 @@ deviceReset()
     # reset is a collection of standard tasks
     sudo /usr/bin/connectd_stop_all
     sudo connectd_control stop all
-    sudo connectd_control reset
+    sudo connectd_control reset "y"
```
Comment (penngrove) [inline connectd/usr/bin/connectd_control:884]: no need for double quotes here around the y

File: test/Interactive/interactive-test.sh [hunk 2]
```diff
@@ -12,10 +12,12 @@ checkForRoot
 # If run with no following parameter, just run the script interactively
 # while capturing the debug output (e.g. for manual testing).
 if [ "$1" = "" ]; then
-    sudo sh -x /usr/bin/connectd_installer 2> debug.txt | tee console.txt
+#    sudo sh -x /usr/bin/connectd_installer 2> debug.txt | tee console.txt
```
Comment (penngrove) [inline test/Interactive/interactive-test.sh:15]: you can remove these comments

File: test/Interactive/interactive-test.sh [hunk 3]
```diff
@@ -12,10 +12,12 @@ checkForRoot
...
+    sudo sh -x /usr/bin/connectd_installer 2> /tmp/debug.txt | tee /tmp/console.txt
 else
 # If run with one following parameter, use that parameter as the name of a keystroke file
 # such as configure-01.key, configure-02.key, remove-all.key
-    sudo sh -x /usr/bin/connectd_installer < "$1" 2> debug.txt | tee console.txt
+#    sudo sh -x /usr/bin/connectd_installer < "$1" 2> debug.txt | tee console.txt
```
Comment (penngrove) [inline test/Interactive/interactive-test.sh:20]: same here

Comment (Digital-Larry) [reply test/Interactive/interactive-test.sh:20]: I'll clean up in next PR, thanks

**Pull request merged**